src/test/java/org/springframework/integration/kafka/outbound/KafkaProducerMessageHandlerTests.java

```diff
@@ -195,7 +196,7 @@ public void testOutbound() throws Exception {
 		assertThat(record).has(key(2));
 		assertThat(record).has(partition(1));
 		assertThat(record.value()).isNull();
-
+		
```

Comment:
I think these are wrong symbols for indentations.
Consider to use only *tabs*

Reply:
This is becoming embarrassing ;-)
I hope everything is OK now.